ctrlf.el

```diff
@@ -569,6 +569,33 @@ non-nil."
 
 ;;;;; Utility functions
 
+(defun ctrlf--set-evil-jump ()
+  "Integration with evil's jump list.
+Adds an entry to evil's jump list for the current position."
+  (when (fboundp 'evil-set-jump)
```

Comment:
Should this also be conditioned on `(bound-and-true-p evil-mode)`?

Reply:
Got it this way off swiper, but you're right, it is better to be consistent.

Reply:
Just noticed that checking evil-mode leads to `evil-set-jump` not being known to be defined by the byte compiler. So it's either back to the `fboundp` or another `with-no-warninigs`. I reverted to the former now.

Reply:
Sorry for the delay. I guess the question I have here, is this going to do the right thing when Evil is installed, but not activated? In other words, is it expected that we still call `evil-set-jump` even when `evil-mode` is not enabled?

Reply:
I am confident that this won't be a problem.

 - Swiper does it like this, and it has supported evil for a while now. It's a popular package, so I think if this approach was wrong it would've been noticed by now.
 - In the code I do no see anything that might lead to problems, it's all marker pushing and setting struct fields.
 - Testing it myself was no problem either. You can still navigate through the jump list that's created by ctrlf even if evil-mode is off. You just have to do it manually because there's no key bindings.

Reply:
Okay! Sounds good to me then.